src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -1871,27 +1871,26 @@ static void prepare_driver_info()
 
 	const std::string str_copyright = DOSBOX_COPYRIGHT;
 
-	auto read_low_nibble_str = [&](const uint8_t byte) {
-		return std::to_string(static_cast<int>(read_low_nibble(byte)));
+	auto low_nibble_str = [&](const int byte) {
+		return std::to_string(low_nibble(byte));
 	};
-	auto read_high_nibble_str = [&](const uint8_t byte) {
-		return std::to_string(static_cast<int>(read_high_nibble(byte)));
+	auto high_nibble_str = [&](const int byte) {
+		return std::to_string(high_nibble(byte));
```

Comment:
Maybe this was your motivation to change this?  I don't think `static_cast` was needed in the original code though.  `uint8_t` to `int` is always safe so should be fine as an implicit cast.

Reply:
Yeah, I think that was it.